lib/AST/Decl.cpp

```diff
@@ -1786,12 +1788,26 @@ bool ExtensionDecl::isConstrainedExtension() const {
   return !typeSig->isEqual(extSig);
 }
 
+/// Is the extension written as an unconstrained extension? This is not the same
+/// as isConstrainedExtension() in the case where the extended nominal type has
+/// inverse requirements, because an extension of such a type introduces default
```

Comment:
What's an example of an inverse requirement?

Reply:
Slava wrote a code example a few lines below this using `~Copyable`